mobilitypath_publisher/CMakeLists.txt

```diff
@@ -0,0 +1,98 @@
+#
+# Copyright (C) 2018-2020 LEIDOS.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+cmake_minimum_required(VERSION 2.8.3)
+project(mobilitypath_publisher)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+add_compile_options(-std=c++11)
+set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
+set( CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall")
+
+## Find catkin macros and libraries
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
+  std_msgs
+  cav_msgs
+  carma_utils
+  tf2_ros
+)
+
+## System dependencies are found with CMake's conventions
+find_package(Boost REQUIRED COMPONENTS system)
+
+###################################
+## catkin specific configuration ##
+###################################
+
+catkin_package(
+  CATKIN_DEPENDS roscpp std_msgs cav_msgs carma_utils tf2_ros
+)
+
+###########
+## Build ##
+###########
+
+include_directories(
+  ${catkin_INCLUDE_DIRS}
+  ${Boost_INCLUDE_DIRS}
+  include
+)
+
+file(GLOB_RECURSE headers */*.hpp */*.h)
+
+add_executable( ${PROJECT_NAME}
+  ${headers}
+  src/mobilitypath_publisher.cpp
+  src/main.cpp)
+
+target_link_libraries(${PROJECT_NAME} mobilitypath_publisher_lib ${catkin_LIBRARIES} ${Boost_LIBRARIES})
+
+add_library(mobilitypath_publisher_lib src/mobilitypath_publisher.cpp)
+
+add_dependencies(mobilitypath_publisher_lib ${catkin_EXPORTED_TARGETS})
+
+
+#############
+## Install ##
+#############
+
+install(DIRECTORY include/${PROJECT_NAME}/
+  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
+  FILES_MATCHING PATTERN "*.hpp"
+  PATTERN ".svn" EXCLUDE
```

Comment:
You don't need 
```
 FILES_MATCHING PATTERN "*.hpp"
  PATTERN ".svn" EXCLUDE
```
Please remove. Another thing that keeps ending up in new packages.

Reply:
Removed second line, but the FILES_MATCHING PATTERN "*.hpp" seems to be needed for install